internal/clients/team6/judge.go

```diff
@@ -16,6 +16,7 @@ func (j *judge) GetPardonedIslands(currentSanctions map[int][]shared.Sanction) m
 
 	for timeStep, sanctions := range currentSanctions {
 		for who, sanction := range sanctions {
+			pardons[timeStep] = make([]bool, len(sanctions))
```

Comment:
```suggestion
                pardons[timeStep] = make([]bool, len(sanctions))
		for who, sanction := range sanctions {
```

Reply:
shouldnt these two lines be swapped around?

Reply:
Soz, brain not working

Reply:
Ohhh just saw this sry for this stupid mistake. Thx for the correction!